model/modeldecoder/v2/model.go

```diff
@@ -317,6 +317,15 @@ type transactionUserExperience struct {
 	// TotalBlockingTime holds the Total Blocking Time (TBT) metric value,
 	// or a negative value if TBT is unknown. See https://web.dev/tbt/
 	TotalBlockingTime nullable.Float64 `json:"tbt" validate:"min=0"`
+
+	// Longtask holds longtask duration/count metrics.
+	Longtask longtaskMetrics `json:"longtask"`
```

Comment:
From how I understand https://github.com/elastic/apm-server/issues/4226#issuecomment-696769130 `longtask` would also be shortened here to `lt`. @jahtalab could you verify please if it is `longtask` or `lt` when sending to the `v2` Intake?

Reply:
@simitt , all compression algorithms (including shortening) only apply to apiVersion > 2, so for intake v2 we send the original document.

Reply:
Does that only apply to `longtask` or also to other measurements for `v2` under `experience`? (Because they _are_ shortened on the v2 Intake, see https://github.com/elastic/apm-server/blob/master/docs/spec/rum_experience.json#L7)

Reply:
We generally prefer to use abbreviations of known metrics, so in the case of cls, that's a known abbreviation of cumulative layout shift, in those case it doesn't make sense to shorten the field any further.